addons/project/static/src/components/subtask_one2many_field/subtask_list_renderer.js

```diff
@@ -4,10 +4,22 @@ import { useService } from "@web/core/utils/hooks";
 import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
 import { ListRenderer } from '@web/views/list/list_renderer';
 
+const { useEffect } = owl;
+
 export class SubtaskListRenderer extends ListRenderer {
     setup() {
         super.setup();
         this.dialog = useService("dialog");
+        useEffect(
+            () => this.focusToName(this.props.list.editedRecord),
+        )
+    }
+
+    focusToName(editRec) {
+        if (editRec?.isNew) {
```

Comment:
```suggestion
        if (editRec?.isNew && !editRec.isDirty) {
```
Imo, a check on `isDirty` is also needed to avoid to focus on the name if the user clicks on a field of a subtask that has already been added to the tree but not yet saved.